meson.build

```diff
@@ -9,9 +9,27 @@ project('gnome-hexgl', 'c',
 
 gthree = dependency('gthree-1.0')
 
+prefix = get_option('prefix')
+datadir = join_paths(prefix, get_option('datadir'))
+
 cc = meson.get_compiler('c')
 libm = cc.find_library('m', required : false)
 
-sources = ['src/hexgl.c', 'src/utils.c', 'src/analysismap.c', 'src/camerachase.c', 'src/shipcontrols.c', 'src/shipeffects.c', 'src/shaders.c', 'src/particles.c', 'src/hud.c']
+sources = [
+  'src/hexgl.c',
+  'src/utils.c',
+  'src/analysismap.c',
+  'src/camerachase.c',
+  'src/shipcontrols.c',
+  'src/shipeffects.c',
+  'src/shaders.c',
+  'src/particles.c',
+  'src/hud.c'
+]
+
+add_project_arguments(['-DDATADIR="@0@"'.format(datadir)], language: 'c')
+
+install_subdir('models', install_dir: datadir)
```

Comment:
We should not install the original *.blend (or any *.blend1 backups) files, only the .glb files created from them. Can we pass a set of globs to install_subdir?

Reply:
there's exlude_files, but do not accept regex, so I will do a proper meson recursive install